subsys/usb/usb_device.c

```diff
@@ -503,18 +513,53 @@ static bool set_endpoint(const struct usb_ep_descriptor *ep_desc)
 		ep_cfg.ep_addr, ep_cfg.ep_type, ep_cfg.ep_mps);
 
 	if (usb_dc_ep_configure(&ep_cfg) < 0) {
-		LOG_WRN("Failed to configure endpoint %x", ep_cfg.ep_addr);
```

Comment:
I changed this and make some smoke tests, and what i see:
**abbreviations**
vm - virtual machine
nm - native machine

I have setup contained on vm (linux) running on nm (win10). When i build and flash the device connected to vm and then disconnect it from vm I can see the those LOG_ERR because the endpoints are already enabled (they were enabled under linux vm).

I think in this case I will leave this as warning, bcs the device enumerates correctly. What is more I removed
```
return false;
```
set_endpoint is called when set_configuration request was received. With this false I was constantly returning false because epts were already configured and this lead to enumeration error.

I am writing this just to keep You aware of this.

Reply:
okay